src/main/java/uk/gov/companieshouse/api/accounts/validation/ProfitAndLossValidator.java

```diff
@@ -38,6 +39,7 @@ public Errors validateProfitLoss(@Valid ProfitAndLoss profitAndLoss, String comp
 
             validateGrossProfitTotal(profitAndLoss.getGrossProfitOrLoss(), errors);
             validateOperatingTotal(profitAndLoss, errors);
+            validateProfitOrLossBeforeTax(profitAndLoss,errors);
             if(errors.hasErrors()) {
```

Comment:
I know you have not added these lines from (43 to 45). Do you think we really need it?